scss/bitstyles/organisms/navbar/_.scss

```diff
@@ -1,4 +1,4 @@
-.c-navbar {
+.o-navbar {
```

Comment:
I guess that means we have to reflect the changes in the projects, right? 

Would it make sense to start writing CHANGELOG entries for things like this (breaking changes at least)

Reply:
Yes totally 👍 Any classname change or removal is a breaking change (though I don’t think we should  bump the major version number). I’ll start using the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) format for the commit messages on merge to main, hopefully that’ll help make sure we don’t miss any when compiling a release. Should I also add a changelog.md file?

Reply:
I think this could be quite nice :) 
M_alte did add one to chromic_pdf and the format is quite good. 
https://github.com/bitcrowd/chromic_pdf/blob/main/CHANGELOG.md

Reply:
Maybe even have a `Breaking` section for breaking changes.